greenplumpython/func.py

```diff
@@ -86,7 +86,21 @@ def apply(self, expand: Optional[bool] = None, as_name: Optional[str] = None) ->
         if expand and as_name is None:
             as_name = "func_" + uuid4().hex
         if not expand and as_name is None:
-            as_name = self.function.unwrap().__name__
+            if isinstance(self.function, AggregateFunction):
+                try:
+                    self.function.transition_function.unwrap()
+                except:
```

Comment:
Transition_function does not contain wrap_func for already existing function

Reply:
Because for python udf, the `qualified_name` will be too complex, and we still need to test i`f wrapped_func is None
```
_name = "func_" + uuid4().hex if wrapped_func is not None else name
qualified_name = (
            f"pg_temp.{_name}"
            if wrapped_func is not None
            else _name
            if schema is None
            else f"{schema}.{_name}"
        )
```